dolphinscheduler-server/src/main/java/org/apache/dolphinscheduler/server/worker/processor/TaskExecuteProcessor.java

```diff
@@ -95,8 +95,6 @@ public TaskExecuteProcessor() {
      * @param taskExecutionContext task
      */
     private void setTaskCache(TaskExecutionContext taskExecutionContext) {
-        TaskExecutionContext preTaskCache = new TaskExecutionContext();
-        preTaskCache.setTaskInstanceId(taskExecutionContext.getTaskInstanceId());
         taskExecutionContextCacheManager.cacheTaskExecutionContext(taskExecutionContext);
```

Comment:
don't delete this

Reply:
should modify taskExecutionContextCacheManager.cacheTaskExecutionContext(preTaskCache);

Reply:
I had ever second situation of you said, but new preTaskCache cached is required when only the id information needs to be cached

Reply:
What is your question?

Reply:
private void setTaskCache(TaskExecutionContext taskExecutionContext) {
        taskExecutionContextCacheManager.cacheTaskExecutionContext(taskExecutionContext);
    }
then call this:
 public void cacheTaskExecutionContext(TaskExecutionContext taskExecutionContext) {
        taskExecutionContextCache.put(taskExecutionContext.getTaskInstanceId(), taskExecutionContext);
    }